MAINTAINERS.yml

```
@@ -3422,6 +3425,7 @@ Renesas R-Car Platforms:
     - boards/arm/rcar_*/
     - boards/arm64/rcar_*/
     - drivers/*/*rcar*
+    - drivers/pinctrl/renesas/rcar/
     - drivers/clock_control/*cpg_mssr*
```

Please sort in alphabetical order (as in file tree)

```suggestion
    - drivers/clock_control/*cpg_mssr*
    - drivers/pinctrl/renesas/rcar/
```

---

Could you please let me know when you finish the review? I am having difficulty understanding whether you have completed the review or just posted suggestions from your partial review.

I'm concerned that after fixing this one cosmetic issue, you may discover a new cosmetic issue in the unchanged code after a few days or weeks

---

Review of changes is finished, it was last few nit-picks, I forgot to stale my "request for changes"

I'm only reviewing changed code.